vms/platformvm/state/camino_dao_proposal.go

```diff
@@ -0,0 +1,230 @@
+package state
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/ava-labs/avalanchego/database"
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/vms/platformvm/blocks"
+	"github.com/ava-labs/avalanchego/vms/platformvm/txs"
+)
+
+const (
+	ProposalContentLength = 1024
+)
+
+type ProposalType uint64
+
+const (
+	NOPProposal ProposalType = iota
+)
+
+type ProposalState uint64
+
+const (
+	Pending ProposalState = iota
+	Active
+	Concluded
+)
+
+type ProposalOutcome uint64
```

Comment:
This only works for a true / false proposal, but flexibility should be maintained to have a winner option.
I would remove this one.

Reply:
Removed